protocol/x/clob/ante/clob.go

```diff
@@ -250,9 +256,93 @@ func IsShortTermClobMsgTx(ctx sdk.Context, tx sdk.Tx) (bool, error) {
 
 	numMsgs := len(msgs)
 	if numMsgs > 1 {
+		return true, errorsmod.Wrap(
+			sdkerrors.ErrInvalidRequest,
+			"a transaction containing short term MsgCancelOrder or MsgPlaceOrder may not contain more than one message",
+		)
+	}
+
+	return true, nil
+}
+
+// hasClobMsg returns `true` if the transaction has a single clob msg
+func HasClobMsg(tx sdk.Tx) bool {
+	msgs := tx.GetMsgs()
+
+	for _, msg := range msgs {
+		switch msg.(type) {
+		case *types.MsgCancelOrder:
+			return true
+		case *types.MsgPlaceOrder:
+			return true
+		case *types.MsgBatchCancel:
+
+			return true
+		}
+	}
+	return false
+}
+
+// IsValidClobMsgTx returns `true` if the transaction contains a valid set of clob msgs
```

Comment:
Specify when
- the method returns error
- the method returns `false` but no error.